test/Microsoft.DocAsCode.MarkdigEngine.Tests/NolocTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Microsoft.DocAsCode.MarkdigEngine.Tests
+{
+    using Xunit;
+
+    public class NolocTest
+    {
+        [Fact]
+        [Trait("Related", "Noloc")]
+        public void NolocTest_General()
+        {
+            // Normal syntax
+            var source1 = "使用 :::noloc text=\"Find\"::: 方法.";
+            var expected1 = "<p>使用 Find 方法.</p>\n";
+            TestUtility.AssertEqual(expected1, source1, TestUtility.MarkupWithoutSourceInfo);
+
+            // Escape syntax
+            var source2 = "使用 :::noloc text=\"Find a \\\"Quotation\\\"\"::: 方法.";
+            var expected2 = "<p>使用 Find a \"Quotation\" 方法.</p>\n";
+            TestUtility.AssertEqual(expected2, source2, TestUtility.MarkupWithoutSourceInfo);
+
+            // Markdown in noloc
+            var source3 = @":::noloc text=""*Hello*"":::";
+            var expected3 = @"<p>*Hello*</p>
```

Comment:
As the expected3 is not `<p><em>hello</em></p>`, the syntax seems more like `nomarkup` instead of `noloc` to me.

Reply:
Yes, this is the current design reviewed by @cillroy. Full details:
* Do not render markdown syntax inside this no-loc syntax.
* Do not write multiple lines inside this no-loc syntax.
* Only lowercase allowed.
* No optional spaces allowed.
* Escaped characters are allowed ONLY in text part, :::noloc text="\"Quotation\""::: will be rendered to "Quotation". But escaped line-break character is still not allowed.
* Escaped characters in other part are not allowed.